test/conformance/kuttl/generate/policy/standard/data/sync/pol-data-sync-delete-trigger/01-manifests.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pol-data-sync-delete-trigger-ns
+---
+apiVersion: v1
```

Comment:
The ConfigMap needs to be created after the Policy is created. I don't see how this would work properly otherwise.